sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/models/EvaluationEvent.java

```diff
@@ -0,0 +1,86 @@
+package com.azure.spring.cloud.feature.management.models;
+
+public class EvaluationEvent {
+
+    private final Feature feature;
+
+    private String user = "";
+
+    private Boolean enabled = false;
+
+    private Variant variant;
+
+    private VariantAssignmentReason reason = VariantAssignmentReason.NONE;
```

Comment:
Should happen automatically. Not opposed to keeping it if you think it's more clear.

Reply:
It is required, or else it is `null` by default.

Reply:
Oh I thought we determined the enum would default to the first one. Nvm then.